x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/index.tsx

```diff
@@ -624,6 +604,11 @@ const RuleDetailsPageComponent: React.FC<DetectionEngineComponentProps> = ({
     [containerElement, onSkipFocusBeforeEventsTable, onSkipFocusAfterEventsTable]
   );
 
+  const isAlertTableLoading = useMemo(
+    () => loading || isAlertsLoading || !Array.isArray(detectionPageFilters),
```

Comment:
There isn't a scenario where `detectionPageFilters` is somehow set to null or undefined at runtime causing the table to be stuck on a loading state? I didn't see any, but just wanted to confirm

Reply:
@michaelolo24 , you are right. They are either empty array or array of Filters.